pkgs/development/tools/build-managers/conan/default.nix

```diff
@@ -1,29 +1,60 @@
-{ lib, buildPythonApplication, fetchPypi
-, requests, fasteners, pyyaml, pyjwt, colorama, patch
-, bottle, pluginbase, six, distro11, pylint, node-semver2
-, future, pygments, mccabe
-, fetchpatch
-}:
-
-buildPythonApplication rec {
-  version = "1.1.1";
+{ lib, fetchpatch, python }:
```

Comment:
Now you can drop `pythonPackages` from `pythonPackages.callPackage` in `all-packages.nix`.